homeassistant/components/influxdb/__init__.py

```diff
@@ -241,10 +250,14 @@ def event_to_json(event: Dict) -> str:
         if _include_value:
             json[INFLUX_CONF_FIELDS][INFLUX_CONF_VALUE] = _state_as_value
 
+        ignore_attributes = set(entity_config.get(CONF_IGNORE_ATTRIBUTES, []))
```

Comment:
Is there already a default due to the config validation?

Reply:
No, can't do that here. Since these are effectively InfluxDB specific customizations for entities it uses the [entity values](https://github.com/home-assistant/core/blob/0d54dc340957a2101977edb42680c1ceb04fb363/homeassistant/helpers/entity_values.py) helper class for processing. That returns an empty dictionary if there are no customizations for the entity specified. So we can't count on config validation to have put in an empty array here, most entities won't have any customizations and `entity_config` will be an empty dictionary.